prow/cmd/branchprotector/protect.go

```diff
@@ -60,6 +62,7 @@ func gatherOptions() options {
 	fs.StringVar(&o.config, "config-path", "", "Path to prow config.yaml")
 	fs.StringVar(&o.jobConfig, "job-config-path", "", "Path to prow job configs.")
 	fs.BoolVar(&o.confirm, "confirm", false, "Mutate github if set")
+	fs.BoolVar(&o.verifyAuthorisation, "verify-request", false, "Verify the restrictions section of the request for authorized collaborators/teams")
```

Comment:
verify-request seems too vague, verify-restrictions or verify-authorization?

Reply:
Thank you for the review. I have updated to `verify-restrictions` (also updated the validation function name to better match its task).